src/Microsoft.Framework.Configuration.File/FileConfigurationbuilderExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Framework.Configuration
+{
+    public static class FileConfigurationBuilderExtensions
```

Comment:
Hmmm, I used to think so, but having both helpers and extensions at the same time in the name  sounds goofy. Maybe the class should be called `FileHelpers` or maybe the package should be called `FileExtensions`... We can wait until we discuss this with Fowler tomorrow, in case we decide to take a completely different approach.